opentreemap/importer/templates/importer/partials/import_table.html

```diff
@@ -11,7 +11,7 @@
     <tbody>
     {% for ie in things %}
         <tr>
-            <td>{{ ie }}</td>
+            <td>{{ ie.created }}</td>
```

Comment:
Shouldn't this be formatted using the instance's date format?

Like so: `{{ ie.create|date:request.instance.date_format }}`

.. Though to be more accurate, we should probably add an instance-level `datetime_format` [here](https://github.com/OpenTreeMap/OTM2/blob/master/opentreemap/treemap/instance.py#L234-L238) and use that, since `date_format` does not include time.